### PR TITLE
Me dpc 3492 keyresource code coverage update

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-3492

## 🛠 Changes

Added unit tests for KeyResource.  Removed KeyResource and TokenResource unit tests from Jacoco exclusion list.

## ℹ️ Context for reviewers

This was done as part of [DPC-3381](https://jira.cms.gov/browse/DPC-3381) which is an attempt to raise code coverage on the resources folder.

## ✅ Acceptance Validation

Code coverage now at 87% for KeyResource.

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/KeyResourceUnitTest.java [hunk 1]
```diff
@@ -50,11 +110,32 @@ public void testSubmitKey() throws GeneralSecurityException, IOException {
...
+    }
+
+    @Test
+    public void testSubmitKeyBadPEMString() {
+        OrganizationPrincipal organizationPrincipal = APITestHelpers.makeOrganizationPrincipal();
+        KeyResource.KeySignature keySignature = new KeyResource.KeySignature("badPEMString", "badSignature");
+
+        WebApplicationException exception =  assertThrows(WebApplicationException.class,
+                () -> resource.submitKey(organizationPrincipal, keySignature, Optional.of("label")));
+        assertEquals(HttpStatus.SC_BAD_REQUEST, exception.getResponse().getStatus());
```
Comment (kyeah) [inline dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/KeyResourceUnitTest.java:139]: Could we also assert on the error message in these two SubmitKey tests if that's possible? :D Just to confirm that it returned an error for the scenario we're testing, and not for a different scenario.

File: pom.xml [hunk 2]
```diff
@@ -653,9 +653,6 @@
                         <exclude>**/*_.class</exclude>
                         <!--Exclude auto-generated JOOQ classes-->
                         <exclude>gov/cms/dpc/attribution/dao/**/*</exclude>
-                        <!-- Classes manually created using the UnitofWorkAwareProxyFactory are not instrumented correctly, so remove them from analysis, otherwise they show 0%-->
-                        <exclude>gov/cms/dpc/api/resources/v1/TokenResource*</exclude>
-                        <exclude>gov/cms/dpc/api/resources/v1/KeyResource*</exclude>
```
Comment (kyeah) [inline pom.xml:658]: Gonna assume the 0% coverage thing is not a problem anymore 🙂

**Review by kyeah — APPROVED** [review]
LGTM overall 👋 No blocking comments. Feel free to merge with the CVE ignore lines and then revert them as part of your other PR to upgrade rails.

File: dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/KeyResourceUnitTest.java (hunk 1 above)
Comment (MEspositoE14s) [reply dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/KeyResourceUnitTest.java:142]: Added asserts for all of the error messages.

**Pull request merged**